docs/overview/script-storage.md

```diff
@@ -38,20 +37,61 @@ To save a copy of a script to SharePoint, go to the **More options (…)** menu
 > [!NOTE]
 > Power Automate supports running scripts stored on SharePoint with the **Run script from SharePoint library (Preview)** action. This action is currently in preview and is subject to change based on feedback. If you encounter any issues with this action, please report them through the **Help** > **Give Feedback** option in Power Automate.
 
+# [For personal/family](#tab/home)
+
+Scripts are stored in your local Office cache. The cache is in the following folder.
+
+```
+%LOCALAPPDATA%\Microsoft\Office\16.0\Wef\
+```
+
+---
+
+## Share scripts
+
+# [For business/education](#tab/business)
+
+To give users who are outside of the SharePoint site access to the script, [share the script with an Excel workbook](excel.md#share-office-scripts). This means you're linking the script with the file, not attaching it. Whoever has access to the Excel file will also be able to view, run, or make a copy of the script.
+
+> [!NOTE]
+> Admin settings for Conditional Access in OneDrive and SharePoint affect Office Scripts. For more information, see the [Conditional Access section of Platform limits and requirements with Office Scripts](../testing/platform-limits.md#conditional-access).
+
+# [For personal/family](#tab/home)
+
+Scripts can't be shared through workbooks with a home or family account. You can manually share the script files, but there are no in-client support options or change notifications.
+
+---
+
 ## Restore deleted scripts
 
+# [For business/education](#tab/business)
+
 When you delete a script in Excel, it goes to your OneDrive or SharePoint recycle bin. To restore a deleted script, follow the steps listed in [How to recover missing, deleted or corrupted items in SharePoint and OneDrive for work or school](https://support.microsoft.com/office/3d748edf-c072-46c9-81a4-4989056ebc87). Restoring an **.osts** file returns it to the **All scripts** list.
 
 A deleted script is unshared with the workbook. When you restore a script, it does **not** retain its script access. You will need to share the script again.
 
 Restored scripts still work as expected with Power Automate flows. You don't need to recreate the flow connector.
 
+# [For personal/family](#tab/home)
+
+When you delete a script in Excel, it goes to your recycle bin. Restoring an **.osts** file returns it to the **All scripts** list.
```

Comment:
Unfortunately, we don't have scripts as actual osts files in consumer (yet).

They're just in the browser localstorage cache as entries there.  When you delete it... it's gone.  One reason why we're marked as "Preview" for now.